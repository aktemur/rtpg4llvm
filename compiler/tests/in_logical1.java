/*expr*/
x && y || z && w || t ;
