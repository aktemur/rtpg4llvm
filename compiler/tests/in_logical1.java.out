mkBinOp(LogicalOr,
         mkBinOp(LogicalOr,
            mkBinOp(LogicalAnd,
               new VarExpr("x"),
               new VarExpr("y")),
            mkBinOp(LogicalAnd,
               new VarExpr("z"),
               new VarExpr("w"))),
         new VarExpr("t"))
